client/src/elements/item-preview/preview-container.css

```diff
@@ -1,26 +1,8 @@
 preview-container {
+  max-width: calc(85vw - 250px);
```

Comment:
I would prefer to have this defined outside of the item-preview component and use 100% width. Then set the width in the layout stylesheet where this component is used.

Reply:
Fixed in https://github.com/nzzdev/Q-editor/pull/119/commits/10bcb8e93029102517eace7d28a4a713e9ab0dbe